vertx/public/dataviewer/src/PropertyViewModel.js

```diff
@@ -107,14 +111,18 @@ module.exports = function(tree, shouter, worker) {
         propertiesElem.show();
         $('.dataValueInput').hide();
         input.hide();
+        var aclModif = (aclOptions.value).replace('$p', 'P');
+        var testACL = (csync.acl)[aclModif];
```

Comment:
This is kind of hackish? If we add more ACLs or mess with groups in the future this will break.